### DISCUSSION TITLE
Embed DeskHopL mode, enable/disable it using dedicated hotkey

### DISCUSSION THREAD

**Discussion by Scalpos** [hrvach/deskhop#161] [Ideas]
Hi.
Would it be possible to have _a la [Dekhop**L**](https://github.com/jalmeroth/deskhopl) mode_ available, toggable by a specific shortkey ? When this mode is activated, there would be no autowitch between computers, it would require Caps Lock to be pressed.
As non-horizontal multi-monitor setup is not (yet ?) supported, it would allow keeping same computer's monitors screens non-aligned and without forcing one to be the primary and on the edge for automatic switching.

**Comment by hrvach** [comment]
That is a good suggestion, I'll also look into vertical monitors soon. Jan's (@jalmeroth) version has many great ideas and I'll try to pull in some of them (of course, providing full credit).

**Comment by jalmeroth** [comment]
I'm wondering isn't that what you suggest @Scalpos actually the gaming mode? You can try it by hitting `LEFT CTRL + RIGHT SHIFT + G`.

It should enable you to use your monitor setup as before and also just switches, when you hit the define hotkey_toggle `LEFT CTRL + CAPS_LOCK`.
